src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -17,6 +17,9 @@
     <Dependency Include="Microsoft.NETCore.Windows.ApiSets">
       <Version>1.0.1-rc3-23915</Version>
     </Dependency>
+    <Dependency Include="Microsoft.NETCore.Jit">
```

Comment:
I added Version since JIT will move out of the repo at somepoint and this change keeps us in that direction and when that happens, we will simply need to remove VersionSuffix (which is same for both since they build together) and place a fixed version.